lib/linguist/languages.yml

```diff
@@ -41,41 +41,41 @@
   type: programming
   color: "#814CCC"
   extensions:
-  - ".bsl"
-  - ".os"
+    - ".bsl"
+    - ".os"
```

Comment:
Looks like these changes were introduced by mistake.

Reply:
Whoops, thanks for pointing that out @pchaigno. Should be fixed now